suro-kafka-producer/src/test/java/com/netflix/suro/sink/kafka/TestKafkaSink.java

```diff
@@ -95,6 +99,49 @@ public Object findInjectableValue(Object valueId, DeserializationContext ctxt, B
         });
     }
 
+    /**
+     * open will success because localhost is a resolvable hostname
+     */
+    @Test
+    public void testOpenWithDownBroker() throws Exception {
```

Comment:
there is no broker running at "localhost:7101" for this test. maybe it will be less confusing if I change "localhost:7101" to "localhost:0"